platform/ios/MGLMapboxEvents.m

```diff
@@ -144,6 +142,8 @@ @interface MGLMapboxEvents () <CLLocationManagerDelegate>
 @property (atomic) NSString *appName;
 @property (atomic) NSString *appVersion;
 @property (atomic) NSString *appBuildNumber;
+@property (atomic) NSNumber *version;
```

Comment:
You can box it, e.g. `@(MGLMapboxMetricsVersion)` every time you stick it in a dictionary.